imagestreams/nodejs-rhel7.yml

```diff
@@ -0,0 +1,22 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: rhoar-nodejs
+  annotations:
+    openshift.io/display-name: Node.js
```

Comment:
I wonder if we want to call it something like RHOAR Node.js? @balunasj what do you think?